recipes/go-gox/conda_build_config.yaml

```diff
@@ -0,0 +1,17 @@
+go_compiler:
+  - go-nocgo
+cgo_compiler:
+  - go-cgo
+target_goos:
+  - linux                      # [linux]
+  - darwin                     # [osx]
+  - windows                    # [win]
+target_goarch:
+  - 386                        # [x86]
+  - amd64                      # [x86_64]
+target_goexe:
+  -                            # [unix]
+  - .exe                       # [win]
+target_gobin:
+  - '${PREFIX}/bin/'           # [unix]
+  - '%PREFIX%\bin\'            # [win]
```

Comment:
These should be part of conda-forge-pinning, c.f. conda-forge/conda-forge-pinning-feedstock#169